File: TransCoder_tokenized_test_set_functions/java/MINIMIZE_SUM_PRODUCT_TWO_ARRAYS_PERMUTATIONS_ALLOWED.java
static int minValue ( int A [ ], int B [ ], int n ) {
  Arrays . sort ( A );
  Arrays . sort ( B );
  int result = 0;
  for ( int i = 0;
  i < n;
  i ++ ) result += ( A [ i ] * B [ n - i - 1 ] );
  return result;
}

File: results/py/MINIMIZE_SUM_PRODUCT_TWO_ARRAYS_PERMUTATIONS_ALLOWED.cpp
int f_filled ( int * A , int * B , int n ) {
  sort ( A ) ;
  sort ( B ) ;
  int result = 0 ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) result += ( A [ i ] * B [ n - i - 1 ] ) ;
  return result ;
}
